checkov/common/images/image_referencer.py

```diff
@@ -0,0 +1,29 @@
+from abc import abstractmethod
+
+import docker
+
+
+class ImageReferencer:
+    @abstractmethod
+    def is_workflow_file(self, f):
+        pass
+
+    @abstractmethod
+    def get_images(self, f):
+        return []
```

Comment:
```suggestion
        pass
```
or remove the `@abstractmethod` decorator